### DISCUSSION TITLE
Add Diamond Stealth 64 VIdeo 2001 PCI card (S3 Trio64V+)

### DISCUSSION THREAD

**Discussion by shitsux420** [86Box/86Box#2898] [Ideas]
I'll start this as a discussion right away if necessary.

**Is your feature request related to a problem? Please describe.**
Only Trio64V+ card is a noname Phoenix card with little information in the internet.

**Describe the solution you'd like**
Add a more famous and most likely better working and more compatible card with actual driver support for Win3.1 and 9x.

**Describe alternatives you've considered**
Use plain Trio64.

**Additional context**
Datasheet:
[571_86c765_trio64vplus.pdf](https://github.com/86Box/86Box/files/10097194/571_86c765_trio64vplus.pdf)
Specsheet:
[571_s3_trio64vplus.pdf](https://github.com/86Box/86Box/files/10097198/571_s3_trio64vplus.pdf)
BIOSes:
[TrioBIOS.zip](https://github.com/86Box/86Box/files/10097201/TrioBIOS.zip)

Information over old Diamond site:
https://web.archive.org/web/19961219233922/http://www.diamondmm.com/products/visualization/stealth64v-2001s/
https://web.archive.org/web/19961220001720/http://www.diamondmm.com/products/visualization/stealth64v-2001s/tech-spec.html

**Comment by shitsux420** [comment]
I though I'd bump this if this would still be of any interest for anyone as this card is still missing from 4.0.0.

**Reply by jriwanek** [reply]
You do not need to bump, We can see discussions, and it will not make anything happen faster. If you'd like a focus put on something, we have a patreon tier for that.

**Reply by shitsux420** [reply]
My apologies, I didn't mean to imply some sort of urgency solving this issue or anything, just that it's noticed in the first place since it was never marked with an action, like giving it an answer or adding it to a workflow or a todo list.

**Comment by TechGuyC** [comment]
> I though I'd bump this if this would still be of any interest for anyone as this card is still missing from 4.0.0.

It is most likely that the version 4.0.1 will still be missing many graphics cards

**Reply by jriwanek** [reply]
Minorversions (x.x.y) don't get new cards or machines, just bugfixes.
